domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -995,6 +1000,10 @@ class ExplorationProgressController @Inject constructor(
       val state = progress.stateDeck.getCurrentState()
       hintHandler.startWatchingForHintsInNewState(state)
       startState(logStartCard = true)
+
+      if (!isRestart) {
+        explorationAnalyticsLogger.logStartExploration()
+      }
```

Comment:
Based on my search, we have the following events related to exploration:
 - open_exploration_activity
 - exit_exploration_context
 - finish_exploration_context
 - resume_exploration_context
 - start_over_exploration_context

It appears that there is no log for when a new exploration is started.

Reply:
I think so far, we have been using open_exploration_activity to know when an exploration has been started, but this is a nice addition so that's okay.